text/0202-events-api.md

```diff
@@ -0,0 +1,184 @@
+# Introducing Events and Logs API
+
+In this OTEP we introduce an Events and Logs API that is based on the OpenTelemetry Log signal. The Events here refer to the independent Events and not to be confused with Span Events which occur only in the context of a span. In OpenTelemetry's perspective Log Records and Events are different names for the same concept - however, there is a subtle difference in how they are represented using the underlying data model that is described below. Although every language has APIs for logs, they are not all capable of creating events. We will describe why the existing Logging APIs are not sufficient for the purpose of creating events.  It will then be evident that we will need an API in OpenTelementry for creating events.
```

Comment:
>Although every language has APIs for logs, they are not all capable of creating events.

I think this sentence can be misinterpreted and is probably unnecessary. Perhaps delete it.

Reply:
Deleted.